spec/system/distribution_system_spec.rb

```diff
@@ -119,6 +119,7 @@
         visit @url_prefix + "/distributions/new"
         select @partner.name, from: "Partner"
         select @storage_location.name, from: "From storage location"
+        sleep 1 # BUG: REMOVE THIS SLEEP
```

Comment:
I think this actually works to fix https://github.com/rubyforgood/human-essentials/issues/4248, but its a very poor hack. Not sure what a better approach is.

Reply:
See my comment there.